src/Mod/Sketcher/Gui/SketcherToolDefaultWidget.cpp

```diff
@@ -157,13 +157,14 @@ bool SketcherToolDefaultWidget::eventFilter(QObject* object, QEvent* event)
             }
         }
     }
-    else if (event->type() == QEvent::FocusOut) {
-        for (int i = 0; i < nParameters; i++) {
-            auto parameterSpinBox = getParameterSpinBox(i);
-
-            if (object == parameterSpinBox) {
-                signalParameterFocusOut(i);
-                break;
+    else if (event->type() == QEvent::KeyPress) {

```

Comment:
I doubt that this is good idea. There are more ways to lose focus than pressing Tab or Return.

Reply:
Using focusout was the previous mistake which is causing the bug.
 This should not happen on all the focus out events. Only when tab pressed.

Reply:
OK. But shouldn't then the signal be renamed?

Reply:
Yes it would be better.